From 249f6a3c1ef76d50a84973eae1d92581d6fc2d3a Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Mon, 13 Jan 2020 11:05:42 -0600
Subject: [PATCH 3/4] drivers: ddr: imx8m: Reduce chatter during startup

Three are a few functions in U-Boot that print the status of
the DDR configuration.  These are silent upstream, and not
really needed unless there is an error.

This patch makes some of 'printf' statements 'debug' statements
and leaves the error messages as 'printf' so any errors are
explicitly shown.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 drivers/ddr/imx8m/ddrphy_train.c | 2 +-
 drivers/ddr/imx8m/helper.c       | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/ddr/imx8m/ddrphy_train.c b/drivers/ddr/imx8m/ddrphy_train.c
index 310099d..1648ff4 100644
--- a/drivers/ddr/imx8m/ddrphy_train.c
+++ b/drivers/ddr/imx8m/ddrphy_train.c
@@ -28,7 +28,7 @@ void ddr_cfg_phy(struct dram_timing_info *dram_timing)
 	/* load the frequency setpoint message block config */
 	fsp_msg = dram_timing->fsp_msg;
 	for (i = 0; i < dram_timing->fsp_msg_num; i++) {
-		printf("DRAM PHY training for %dMTS\n", fsp_msg->drate);
+		debug("DRAM PHY training for %dMTS\n", fsp_msg->drate);
 		/* set dram PHY input clocks to desired frequency */
 		ddrphy_init_set_dfi_clk(fsp_msg->drate);
 
diff --git a/drivers/ddr/imx8m/helper.c b/drivers/ddr/imx8m/helper.c
index e50e422..4be7a82 100644
--- a/drivers/ddr/imx8m/helper.c
+++ b/drivers/ddr/imx8m/helper.c
@@ -58,7 +58,7 @@ void ddr_load_train_firmware(enum fw_type type)
 		i += 4;
 	}
 
-	printf("check ddr_pmu_train_imem code\n");
+	debug("check ddr_pmu_train_imem code\n");
 	pr_from32 = imem_start;
 	pr_to32 = DDR_TRAIN_CODE_BASE_ADDR + 4 * IMEM_OFFSET_ADDR;
 	for (i = 0x0; i < IMEM_LEN; ) {
@@ -80,7 +80,7 @@ void ddr_load_train_firmware(enum fw_type type)
 		printf("check ddr_pmu_train_imem code pass\n");
 	}
 
-	printf("check ddr_pmu_train_dmem code\n");
+	debug("check ddr_pmu_train_dmem code\n");
 	pr_from32 = dmem_start;
 	pr_to32 = DDR_TRAIN_CODE_BASE_ADDR + 4 * DMEM_OFFSET_ADDR;
 	for (i = 0x0; i < DMEM_LEN;) {
-- 
2.17.1

